### PR TITLE
feat: #505 new strategy

### PR DESCRIPTION
**Pull request by HuijunDev**
feat(strategy): Add HTML and partial CSS for new strategy page

This PR introduces the initial HTML structure and partial CSS styling for the new strategy page. The main layout elements and essential classes have been defined to provide a foundation for further development. 

- Added HTML structure for the new strategy page 
- Implemented initial CSS for layout and styling 
- Updated dependencies in `package.json` 

strategy

### CODE REVIEW

**Comment by Aria622** [comment]
Looks all good.

File: src/app/layout.tsx [hunk 1]
```diff
@@ -60,12 +60,11 @@ export default function RootLayout({
         <nav className="relative">
           <div
             onChange={handleScroll}
-            className={`fixed z-20 top-0 left-0 right-0 bg-black ${
-              scrolling ? 'bg-opacity-30' : ''
-            } h-[60px] px-2 md:px-8 py-3 text-white`}
+            className={`fixed z-20 top-0 left-0 right-0 bg-black ${scrolling ? 'bg-opacity-30' : ''
+              } h-[60px] px-2 md:px-8 py-3 text-white`}
```
Comment (Yingxue0323) [inline src/app/layout.tsx:64]: why did you delete the scrolling part?

File: src/styles/globals.css [hunk 2]
```diff
@@ -18,19 +19,17 @@
...
-      rgb(var(--background-end-rgb))
-    )
-    rgb(var(--background-start-rgb));
-    margin: 0;
-    padding: 0;
-  
+      rgb(var(--background-end-rgb))) rgb(var(--background-start-rgb));
+  margin: 0;
+  padding: 0;
+
```
Comment (Yingxue0323) [inline src/styles/globals.css:27]: Better to keep the original format, it's more readable

File: src/app/layout.tsx (hunk 1 above)
Comment (HuijunDev) [reply src/app/layout.tsx:64]: > why did you delete the scrolling part?

It was not deleted. A plugin automatically adjusted the format. I will disable the plugin to maintain consistency with everyone.
